test/message/core_line_numbers.out

```diff
@@ -11,5 +11,5 @@ RangeError: Invalid input
     at Module.load (module.js:*:*)
     at Function.Module._load (module.js:*:*)
     at Function.Module.runMain (module.js:*:*)
-    at startup (node.js:*:*)
-    at node.js:*:*
+    at startup (bootstrap_node.js:*:*)
+    at bootstrap_node.js:*:*
```

Comment:
Yes, `ExecuteString` calls `v8::Script::Compile` which allows us to specify the ScriptOrigin, which comes from the FIXED_ONE_BYTE_STRING I changed. `grep ExecuteString ./src/* -r` says it isn't called anywhere else, perhaps `ExecuteString` should be merged into `LoadEnvironment`?